kuzzleio/sdk-javascript
# add getting started for reactJS

## What does this PR do?

This PR add the reactJS getting started project with cypress tests and add the snippets and the Readme of the ReactJS getting started.
Like the others getting started this a simple chat with document create and realtime subscription.




### How should this be manually tested?


  - Step 0 : cd ./doc/7/getting-started/.react/standalone/
  - Step 1 : get a Kuzzle running on localhost
  - Step 2 : npm install
  - Step 3 : npm start
  - Step 4 : npm test

Check Netlify or the /doc/7/getting-started/react/standalone//index.md

### Other changes



### Boyscout
